demos/gallery/samples/common/TitleBar.js

```diff
@@ -0,0 +1,32 @@
+import { TitleBar, SelectionButton, Spacer } from "@hpcc-js/common";
+/*
+Button.prototype.publish("gutter", 2, "number", "Butting padding and margin (pixels)");
+Button.prototype.publish("faChar", "", "string", "FontAwesome character (if done right will look like a  in your favorite editor)");
+Button.prototype.publish("faClass", "", "string", "FontAwesome class name (if done right will look like fa-user)");
+Button.prototype.publish("label", "", "string", "The label/text written on the button (to the right of the faChar (if a faChar is provided))");
+Button.prototype.publish("lastFAChar", "", "string", "The 'last-icon' displays on the far right face of the button. FontAwesome character (if done right will look like a  in your favorite editor)");
+Button.prototype.publish("lastFAClass", "", "string", "The label/text written on the button (to the left of the 'last-icon' (if a faChar is provided)). The 'last-icon' displays on the far right face of the button.");
+Button.prototype.publish("tooltip", "", "string", "Displays as the button alt text attribute");
+Button.prototype.publish("state", "enabled", "set", "Current 'state' of the button is usually 'enabled'.", ["enabled", "disabled", "selected"]);
+Button.prototype.publish("styleMode", "default", "set", "Several different base button styles.", ["default", "Bootstrap1", "Bootstrap2", "Bootstrap3", "Bootstrap4", "Primer"]);
+*/
```

Comment:
Remove commented code (you could add a comment with a link to the button sources if you thought it was a good idea?)